### PR TITLE
Issues 218 240 244 - Show Quarkus version in Getting Started page

### LINKED ISSUE #218: Offer on the home page or somewhere else the latets release version and link to the shownotes

### PR DESCRIPTION
**Pull request by emmanuelbernard**
Clarify JDK version supported
Clarify GraalVM version
Clarify GraalVM is optional
Clarify IDE choices

Fixes #218 #240 #244

### CODE REVIEW

File: _data/versions.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+quarkus:
+  version: 0.18.0
+  announce: /blog/quarkus-0-18-released/
+graalvm: 19.0.2+
+jdk: "8 to 11"
+maven: 3.5.3+
```
Comment (rsvoboda) [inline _data/versions.yaml]: I thought we require 3.5.4+ due to changes @aloubyansky worked on.

**Comment by emmanuelbernard** [comment]
If that's the case then that's a fail from code change to doc propagation
because the website says 3.5.3

**Comment by rsvoboda** [comment]
Our .adoc files in main Quarkus say `3.5.3+` so the docs propagation process is not broken, probably our .adoc files need update. Talking about it on Zulip now.

**Comment by rsvoboda** [comment]
mvn 3.5.3 worked well on both quarkus and the quickstarts

My motivation for this fuzz was the fact that https://github.com/quarkusio/quarkus/blob/master/bom/pom.xml#L60 says `<maven-core.version>3.5.4</maven-core.version>`

File: _data/versions.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+quarkus:
+  version: 0.18.0
+  announce: /blog/quarkus-0-18-released/
+graalvm: 19.0.2+
+jdk: "8 to 11"
```
Comment (gsmet) [inline _data/versions.yaml]: It's a detail but I would say `8 or 11`.

This makes it clear that we only test and support the LTS versions.

**Pull request merged**